draft-ietf-taps-interface.md

```diff
@@ -1502,6 +1097,732 @@ does provide the following guarantees about the ordering of operations:
+
+The identification of a specific Provisioning Domain (PvD) is defined to be implementation-
+and system-specific, since there is not a portable standard format for a PvD identitfier.
+For example, this identifier may be a string name or an integer. As with
+requiring specific interfaces, requiring a specific PvD strictly limits path selection.
+
+Categories or types of PvDs are also defined to be implementation- and system-specific.
+These may be useful to identify a service that is provided by a PvD. For example, if an application
+wants to use a PvD that provides a Voice-Over-IP service on a Cellular network, it can use
+the relevant PvD type to require some PvD that provides this service, without needing to
+look up a particular instance. While this does restrict path selection, it is more broad than
+requiring specific PvD instances or interface instances, and should be preferred over those
+options.
+
+
+### Capacity Profile {#prop-cap-profile}
+
+Classification: 
+: Intent
+
+Type: 
+: Enumeration
+
+Applicability: 
+: Preconnection, Connection, Message 
+
+This property specifies the application's expectation of the dominating traffic pattern for this Connection.
+This implies that the transport system should optimize for the capacity profile specified. This can influence path and protocol selection.
+The following values are valid for the Capacity Profile:
+
+  Default:
+  : The application makes no representation about its expected
+  capacity profile. No special optimizations of the tradeoff between
+  delay, delay variation, and bandwidth efficiency should be made when selecting and
+  configuring stacks.
+
+  Low Latency:
+  : Response time (latency) should be optimized at
+  the expense of bandwidth efficiency and delay variation when sending this
+  message. This can be used by the system to disable the coalescing of
+  multiple small Messages into larger packets (Nagle's algorithm); to prefer
+  immediate acknowledgment from the peer endpoint when supported by the
+  underlying transport; to signal a preference for lower-latency, higher-loss
+  treatment; and so on.
+
+  Constant Rate:
+  : The application expects to send/receive data at a constant rate after
+  Connection establishment. Delay and delay variation should be minimized at the
+  expense of bandwidth efficiency. This implies that the Connection may fail
+  if the desired rate cannot be maintained across the Path. A transport
+  may interpret this capacity profile as preferring a circuit breaker
+  {{?RFC8084}} to a rate-adaptive congestion controller.
+
+  Scavenger/Bulk:
+  : The application is not interactive. It expects to send/receive a large
+  amount of data, without any urgency. This can, for example, be used to select protocol
+  stacks with scavenger transmission control, to signal a preference for
+  less-than-best-effort treatment, or to assign the traffic to a lower-effort service.
+
+
+### Congestion control {#prop-cc}
```

Comment:
This seems potentially dangerous.

Reply:
What?

Reply:
This has been in the draft for since trammell-taps-iterface-00 - nothing we added